tox.ini

```diff
@@ -125,6 +125,8 @@ deps =
 conda_env = sunpy-dev-env.yml
 install_command = pip install --no-deps --no-build-isolation {opts} {packages}
 allowlist_externals = conda
+# Remove pytest-rerunfailures to avoid appearing to access the internet when pytest-xdist is used
```

Comment:
Can we not move this into the command block or will tox error?

Reply:
I don't know: I still don't run tox locally, and I don't see guidance in the tox documentation.  Maybe it'd work fine?

Reply:
Want me to try?